format/testdata/require_before.clj

```diff
@@ -1,11 +1,13 @@
 (ns foo.bar
-  (:require ; z
-            (z :as z) ; another
-            ; a
-            [a :refer [x y z]] ; blah
-            ; after
-            )
-  (:import java.io.File
-           [java.util.zip GZIPInputStream]
-           java.io.DataInputStream
-           ))
+  (:require
+    ; z
+    [z :as z] ; another
+    ; a
+    [a :refer [x y z]] ; blah
+    ; after
+    )
+  (:import
+    java.io.File
```

Comment:
"How to ns" recommends using a list even if only one class is imported (https://stuartsierra.com/2016/clojure-how-to-ns.html#prefix-lists) -- WDYT about implementing that as well?

Reply:
This is easy enough to implement. The reason I didn't do that is that is because once you change from

```
(:import
  a.X
  a.Y)
```

to

```
(:import
  (a X)
  (a Y))
```

this introduces a duplicate prefix style issue which didn't exist before. Fixing this properly by rewriting to

```
(:import
  (a X Y))
```

takes us back to all the hard problems I described in https://github.com/cespare/goclj/pull/85#issuecomment-777754824.

Of course, in most cases swapping a symbol for a list won't introduce duplicate prefixes, and we can always notice and fix it by hand. Do you think it'd be better to add in this list-ification even if we don't do the deduplication?

Reply:
Ah yeah, I hadn't thought of that case. I'd say it's still worthwhile even without the deduping, since it makes the list of imports read more constantly.

Reply:
OK, I'll add it in.